### PR TITLE
Document how to migrate YAML data to JSON

### PR DESCRIPTION
**Pull request by seanlinsley**
https://github.com/airblade/paper_trail/pull/662#issuecomment-161348466

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1111,6 +1111,57 @@ alter column object type json
...
+  version.update_column :new_object, YAML.load(version.object)
+end
+
+remove_column :versions, :object
+rename_column :versions, :new_object, :object
+```
+
+There's a big downside with this: it can be very slow if you have a lot of data.
+Note that the table will be locked during the migration, so it effictively blocks
+any data modification on tables that use PaperTrail.
```
Comment (jaredbeck) [inline README.md]: In postgres, I don't think it's a table lock, I think it's a transaction.

File: README.md [hunk 2]
```diff
@@ -1111,6 +1111,57 @@ alter column object type json
...
+
+If it's too slow for your needs, and you're okay with any admin UI missing
```
Comment (jaredbeck) [inline README.md]: Admin UIs are not the only use case for PT. Please delete the word "admin".

**Comment by jaredbeck** [comment]
This is very helpful, thanks!

**Comment by jaredbeck** [comment]
Sean, I took the liberty of rewriting this in https://github.com/airblade/paper_trail/commit/c3ec7a84289b7db37cca42bd8ee3006128980de9, just trying to clarify a few things.  Please let me know if I've missed anything, or dropped some part you wanted to keep. Thanks again for writing this up. It will be helpful to people, I'm sure.

**Comment by batter** [comment]
Cheers to both of you! :beers:

**Comment by seanlinsley** [comment]
No worries @jaredbeck, thanks for merging this :smile_cat:

**Pull request merged**